lib/.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```

Comment:
Why did you do this? Doesn't this break things for people that cannot access https://registry.npmjs.org?

Reply:
If using [nrm](https://github.com/Pana/nrm), running `npm i` will override `registry.npmjs.org` in `package-lock.json`